onyo/lib/commands.py

```diff
@@ -401,16 +402,21 @@ def onyo_edit(inventory: Inventory,
     paths
         Paths of assets to edit.
     message
-        A custom commit message.
+        Commit message to append to the auto-generated message.
+    auto_message
+        If `True`, do auto-generate a commit message subject line.
+        The default can be configured via 'onyo.commit.auto-message'.
+        Onyo configures this to be `True` when initializing a repository.
```

Comment:
I don't think this should be commented here. It can change at any time, and doesn't affect how this function works.